install_config/router/default_haproxy_router.adoc

```diff
@@ -1498,18 +1498,45 @@ To verify the actual amount of ARP entries for IPv4, run the following:
 ----
 
 If the number begins to approach the `net.ipv4.neigh.default.gc_thresh3`
-threshold, increase the values. Run the following on the nodes running the
-router pods. The following sysctl values are suggested for clusters with large
-numbers of routes:
+threshold, increase the values. Get the current value by running:
 
 ----
-net.ipv4.neigh.default.gc_thresh1 = 8192
-net.ipv4.neigh.default.gc_thresh2 = 32768
-net.ipv4.neigh.default.gc_thresh3 = 65536
+# sysctl net.ipv4.neigh.default.gc_thresh1
+net.ipv4.neigh.default.gc_thresh1 = 128
+# sysctl net.ipv4.neigh.default.gc_thresh2
+net.ipv4.neigh.default.gc_thresh2 = 512
+# sysctl net.ipv4.neigh.default.gc_thresh3
+net.ipv4.neigh.default.gc_thresh3 = 1024
 ----
 
-To make these settings permanent across reboots, create a
-link:https://access.redhat.com/solutions/1305833[custom tuned profile].
+The following sysctl sets the variables to the {product-title} current default values.
+
+----
+# sysctl net.ipv4.neigh.default.gc_thresh1=8192
+# sysctl net.ipv4.neigh.default.gc_thresh2=32768
+# sysctl net.ipv4.neigh.default.gc_thresh3=65536
+----
+
+To make these settings permanent across reboots, until an {product-title} upgrade 
+occurs, edit the following file on each node:
+
+----
+/usr/lib/tuned/atomic-openshift-node-host/tuned.conf
```

Comment:
@eparis 
man tuned.conf
DESCRIPTION
       This  man page documents format of Tuned 2.0 profile definition files. Profile is stored in /etc/tuned/<profile_name>/tuned.conf or in
       /usr/lib/tuned/<profile_name>/tuned.conf file where the /etc/tuned/ directory has higher priority.